src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -385,18 +385,23 @@ protected string GetUniqueTexlRuntimeName(string suffix, bool suppressAsync = fa
             return char.ToLowerInvariant(name[0]).ToString() + name.Substring(1) + suffix + (IsAsync && !suppressAsync ? "Async" : string.Empty);
         }
 
-        public virtual bool CheckInvocation(TexlBinding binding, TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
+        public virtual bool CheckSemantics(TexlBinding binding, TexlNode[] args, DType[] argTypes, IErrorContainer errors)
```

Comment:
>CheckSemantics [](http://example.com/codeflow?start=28&length=14)

Can you comment on the new invariants?
What's CheckSemantics vs CheckTypes?

Reply:
Added a comment